src/main/java/com/flowpowered/nbt/regionfile/SimpleRegionFileReader.java

```diff
@@ -35,6 +35,7 @@
 import com.flowpowered.nbt.Tag;
 import com.flowpowered.nbt.stream.NBTInputStream;
 
+@Deprecated
```

Comment:
We can also delete most of the code in this class! :D
Just make it wrap and or use the new one.

Reply:
Bah, this code is so ugly I do not really want to touch this :D 
It does not even handle missing chunks or exceptions and has no documentation.

Reply:
All the more reason to get rid of it I would think. An easy one line of code could replace almost that entire file. Doesn't it make make you feel good when you remove lots code?

Maybe I just like seeing code get deleted. :P

As of now this PR adds 1530 new lines of code and only removes 46.